services/web-app/pages/index.js

```diff
@@ -18,270 +20,305 @@ import {
+        href="/libraries/carbon-react"
+        title="Carbon React library"
+        description="Build user interfaces with core components using Carbon's primary library."
       >
-        <Image
-          alt={'image'}
-          src={ChartImg}
-          layout={isLg ? 'responsive' : 'fill'}
-          objectFit="cover"
-        />
+        <ArtDirection>
+          <Image alt={'image'} src={ReactImgSm} layout="responsive" objectFit="cover" />
+          <Image alt={'image'} src={ReactImgMd} layout="responsive" objectFit="cover" />
+          <Image alt={'image'} src={ReactImgMax} layout="responsive" objectFit="cover" />
+        </ArtDirection>
       </FeatureCard>
-      <Grid className={styles['highlight-grid']}>
-        <Column sm={4} md={8} lg={8} xlg={7}>
-          <h2 className={clsx(styles.subheading, styles['subheading--no-padding'])}>
-            Featured libraries
-          </h2>
-          <p className={styles['subheading-content']}>
-            Libraries are the means to contribute, install, and use one or many assets in products
-            and digital experiences.
-          </p>
-        </Column>
-      </Grid>
       <FeatureCard
-        href="/libraries/carbon-react"
-        title="Carbon React library"
-        description="A library of reusable React components to build websites and user interfaces."
+        href="/libraries/ibmdotcom-web-components"
+        title="IBM.com Web Components"
+        description="Create web experiences that adhere to IBM's web standards using this primary component library."
       >
-        <Image
-          alt={'image'}
-          src={ReactImg}
-          layout={isLg ? 'responsive' : 'fill'}
-          objectFit="cover"
-        />
+        <ArtDirection>
+          <Image alt={'image'} src={DotComImgSm} layout="responsive" objectFit="cover" />
+          <Image alt={'image'} src={DotComImgMd} layout="responsive" objectFit="cover" />
+          <Image alt={'image'} src={DotComImgMax} layout="responsive" objectFit="cover" />
+        </ArtDirection>
       </FeatureCard>
-      <Grid className={clsx(styles['highlight-grid'], styles.border)}>
-        <Column sm={4} md={8} lg={8} xlg={7}>
-          <p className={clsx(styles['subheading-large'])}>
-            By standardizing our assets and surfacing them, we can help our makers find assets that{' '}
-            <strong>comply</strong> with platform requirements, are <strong>convenient</strong> to
-            implement, and are <strong>consistent</strong> with design patterns across the company.
-          </p>
-        </Column>
-      </Grid>
+
+      <Divider />
+
+      <P className={styles['intro-paragraph']}>
+        By standardizing and surfacing our assets, the new site helps makers Find assets that{' '}
+        <mark>comply</mark> with platform requirements, are <mark>convenient</mark> to implement,
+        and are <mark>consistent</mark> with design patterns across the company.
+        <br />↓
+      </P>
```

Comment:
this paragraph column span doesn't seem the same as the redlines at md and >lg breakpoints 
<img width="375" alt="image" src="https://user-images.githubusercontent.com/40550942/185988543-9f4b8c1d-9a4f-4020-bbfa-26e232608dc6.png">
<img width="870" alt="image" src="https://user-images.githubusercontent.com/40550942/185988609-05eb9ada-bc04-4b65-9410-60fc9dce0f01.png">

Reply:
Same happens with the first page paragraph

Reply:
Good catch, I'm checking with design to make sure this page should deviate from our standard column widths.

Reply:
Confirmed column width with Jeannie, she updated Figma. Should be good 👍